src/platforms/javascript/common/profiling/index.mdx

```diff
@@ -16,9 +16,11 @@ Browser Profiling is currently in beta. Beta features are still in progress and
 
 </Note>
 
-Because our browser profiling integration is built on top of the profiler exposed by the [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/), it's in beta and will likely only move out once the official spec progresses and gains adoption. As with any beta package, there are risks involved in using it - see platform [status](https://chromestatus.com/feature/5170190448852992)
+Browser profiling integration is built using [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/) and will likely only move out once the specification progresses and gains adoption. See platform [status](https://chromestatus.com/feature/5170190448852992).
 
-Please note, that since profiling API is currently only implemented in Chromium based browsers, the profiles collected will inherently be biased towards that demographic. This is something you'll need to consider if you're basing your decisions on the data collected. We hope that as the API gains adoption, other browsers will implement it as well. If you find browser profiling feature helpful and would like to see it gain further adoption, please consider supporting the spec at the official [WICG repository](https://github.com/WICG/js-self-profiling).
+Note: since profiling API is currently only exposed in Chromium, profiles collected only include that demographic. This is obvious, but should not be forgotten when analyzing the data collected. We hope that as the API gains adoption, other browsers will implement it as well.
```

Comment:
```suggestion
Note that since the profiling API is currently only exposed in Chromium, profiles collected only include that demographic. This is obvious, but should not be forgotten when analyzing the data collected. We hope that as the API gains adoption, other browsers will implement it as well.
```

Reply:
> The profiling API is currently only exposed in Chromium, therefore profiles collected only include that demographic. We hope that as the API gains adoption, other browsers will implement it as well.

Would this be enough? Just looking for ways to shave words off

Reply:
I agree with @smeubank's recommendation. I prefer the shorter version. Also, I think this page would flow better if this note was moved to the bottom of the "Why profiling" block of text. It's important, but not so important that it should be the first thing users read :)